client/post-editor/editor-media-advanced/index.jsx

```diff
@@ -2,42 +2,62 @@
  * External depencencies
  */
 import React, { PropTypes } from 'react';
-import { bindActionCreators } from 'redux';
 import { connect } from 'react-redux';
 
 /**
  * Internal dependencies
  */
-import { toggleEditorMediaAdvanced } from 'state/ui/editor/media/actions';
+import { setEditorMediaEditItem } from 'state/ui/editor/media/actions';
 import Dialog from 'components/dialog';
 
-function EditorMediaAdvanced( { visible, toggleVisible } ) {
-	return (
-		<Dialog isVisible={ visible } onClose={ toggleVisible }>
-			WIP
-		</Dialog>
-	);
-}
+const EditorMediaAdvanced = React.createClass( {
+	propTypes: {
+		item: PropTypes.object,
+		resetEditItem: PropTypes.func,
+		insertMedia: PropTypes.func
+	},
+
+	getDefaultProps() {
+		return {
+			resetEditItem: () => {},
+			insertMedia: () => {}
```

Comment:
>I think it works well when we use `noop` from **lodash** (or any standard noop).

Yeah, I don't feel strongly either way, though I find it's nice to limit the dependencies, and this should become a common enough pattern for an empty function that I don't think it really harms readability.